Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml

```diff
@@ -787,6 +787,7 @@
 						   behavior:TreeViewBehavior.EnableSelectedItemBinding="True"
 						   behavior:TreeViewBehavior.SelectedItem="{Binding SelectedContent, Mode=TwoWay}"
 						   behavior:TreeViewBehavior.AutoToggleItemExpansion="True"
+						   behavior:TreeViewBehavior.DoubleClickCommand="{Binding DoubleClickCommand}"
```

Comment:
Double-clicking can trigger unintentionally. Wouldn't it make more sense to put this in a r-click menu with "save as ..."?
![image](https://user-images.githubusercontent.com/2359550/129435745-467d63cc-43ba-44c5-b016-e7b30652e775.png)

Reply:
Yes, that it also possible. But right clicks to open custom context menus aren't very common on the web, that's why I went for  double click. (It is also used to download a package from the feed, so there is a little symmetry to that.)